qa/tasks/cephfs/test_scrub_checks.py

```diff
@@ -141,7 +139,7 @@ def test_scrub_pause_and_resume_with_abort(self):
 
         # resume and verify
         self._resume_scrub(0)
-        out_json = self._get_scrub_status()
+        out_json = self.fs.get_scrub_status()
         self.assertTrue("no active" in out_json['status'])
```

Comment:
I got the below error. 

```
2021-01-11T12:36:51.345+0000 7f7b217fa700 -1 client.6635 failed to remount (to trim kernel dentries): return code = 1
ceph-fuse[20000]: fuse finished with error 0 and tester_r 1
2021-01-11 12:41:23,210.210 INFO:__main__:Stopped test: test_scrub_pause_and_resume_with_abort (tasks.cephfs.test_scrub_checks.TestScrubControls) in 296.49699s
2021-01-11 12:41:23,211.211 INFO:__main__:test_scrub_pause_and_resume_with_abort (tasks.cephfs.test_scrub_checks.TestScrubControls) ... FAIL
2021-01-11 12:41:23,211.211 INFO:__main__:
2021-01-11 12:41:23,211.211 INFO:__main__:======================================================================
2021-01-11 12:41:23,211.211 INFO:__main__:FAIL: test_scrub_pause_and_resume_with_abort (tasks.cephfs.test_scrub_checks.TestScrubControls)
2021-01-11 12:41:23,211.211 INFO:__main__:----------------------------------------------------------------------
2021-01-11 12:41:23,211.211 INFO:__main__:Traceback (most recent call last):
2021-01-11 12:41:23,211.211 INFO:__main__:  File "/home/jcollin/ceph/qa/tasks/cephfs/test_scrub_checks.py", line 143, in test_scrub_pause_and_resume_with_abort
2021-01-11 12:41:23,212.212 INFO:__main__:    self.assertTrue("no active" in out_json['status'])
2021-01-11 12:41:23,212.212 INFO:__main__:AssertionError: False is not true
2021-01-11 12:41:23,212.212 INFO:__main__:
Cannot find device "ceph-brx"
2021-01-11 12:41:23,229.229 INFO:__main__:
2021-01-11 12:41:23,230.230 INFO:__main__:----------------------------------------------------------------------
2021-01-11 12:41:23,230.230 INFO:__main__:Ran 1 test in 296.498s
2021-01-11 12:41:23,230.230 INFO:__main__:
2021-01-11 12:41:23,230.230 INFO:__main__:FAILED (failures=1)
2021-01-11 12:41:23,230.230 INFO:__main__:
2021-01-11 12:41:23,230.230 INFO:__main__:
2021-01-11 12:41:23,230.230 INFO:__main__:======================================================================
2021-01-11 12:41:23,230.230 INFO:__main__:FAIL: test_scrub_pause_and_resume_with_abort (tasks.cephfs.test_scrub_checks.TestScrubControls)
2021-01-11 12:41:23,230.230 INFO:__main__:----------------------------------------------------------------------
2021-01-11 12:41:23,230.230 INFO:__main__:Traceback (most recent call last):
2021-01-11 12:41:23,230.230 INFO:__main__:  File "/home/jcollin/ceph/qa/tasks/cephfs/test_scrub_checks.py", line 143, in test_scrub_pause_and_resume_with_abort
2021-01-11 12:41:23,231.231 INFO:__main__:    self.assertTrue("no active" in out_json['status'])
2021-01-11 12:41:23,231.231 INFO:__main__:AssertionError: False is not true
2021-01-11 12:41:23,231.231 INFO:__main__:
2021-01-11 12:41:23,231.231 INFO:__main__:
2021-01-11 12:41:23,231.231 INFO:__main__:
```

Reply:
Needs little bit more waiting since little after the test `scrub status` printed "no active scrubs running" -

```
$ ./bin/ceph tell mds.b scrub status
2021-01-12T16:48:28.036+0530 7fd902f25700 -1 WARNING: all dangerous and experimental features are enabled.
2021-01-12T16:48:28.063+0530 7fd902f25700 -1 WARNING: all dangerous and experimental features are enabled.
2021-01-12T16:48:28.099+0530 7fd8daffd700  0 client.4655 ms_handle_reset on v2:192.168.0.218:6836/1167271452
2021-01-12T16:48:28.140+0530 7fd8daffd700  0 client.4662 ms_handle_reset on v2:192.168.0.218:6836/1167271452
{
    "status": "no active scrubs running",
    "scrubs": {}
}
```

Reply:
Okay, tested serval times and locally it worked for me.

Maybe we should retry and wait 30 seconds here ?

Reply:
This is a known issue, please see https://tracker.ceph.com/issues/48812.

Reply:
This doesn't appear any more in my tests.

```
2021-03-05 06:01:33,378.378 INFO:__main__:test_scrub_pause_and_resume_with_abort (tasks.cephfs.test_scrub_checks.TestScrubControls) ... ok
2021-03-05 06:01:33,378.378 INFO:__main__:
2021-03-05 06:01:33,378.378 INFO:__main__:----------------------------------------------------------------------
2021-03-05 06:01:33,378.378 INFO:__main__:Ran 1 test in 231.479s
2021-03-05 06:01:33,378.378 INFO:__main__:
2021-03-05 06:01:33,378.378 INFO:__main__:OK

```